pom.xml

```diff
@@ -3233,6 +3247,10 @@
       <properties>
         <scala.version>2.13.4</scala.version>
         <scala.binary.version>2.13</scala.binary.version>
+        <silencer-plugin.version>1.7.1</silencer-plugin.version>
+        <!-- SPARK-33560: Use a fake globalFilters because silencer plugin is not required since Scala 2.13.2 -->
+        <silencer-plugin.globalFilters>-P:silencer:globalFilters=.*fakeGlobalFilters.*</silencer-plugin.globalFilters>
```

Comment:
Use `-P:silencer:globalFilters=` will suppress all warn, try to make a fake config